pymongo/command_cursor.py

```diff
@@ -239,31 +249,32 @@ def alive(self):
         return bool(len(self.__data) or (not self.__killed))
 
     @property
-    def cursor_id(self):
+    def cursor_id(self) -> int:
```

Comment:
I think this is actually a bson.Int64. Can you confirm?

Reply:
There are places where it is explicitly set to `0`, and `Int64` derives from `int`, I think returning `int` is accurate.